packages/edit-site/src/components/welcome-guide/styles.js

```diff
@@ -119,7 +119,7 @@ export default function WelcomeGuideStyles() {
 								) }
 								<ExternalLink
 									href={ __(
-										'https://wordpress.org/support/article/styles-overview/'
+										'https://wordpress.org/documentation/article/block-based-widgets-editor/'
```

Comment:
Looks like wrong link.

```suggestion
										'https://wordpress.org/documentation/article/styles-overview/'
```

Reply:
Oops, thanks for finding the mistake!